src/units.cpp

```diff
@@ -206,10 +221,11 @@ bool updateUnitMultipliers(double &multiplier,
             mult = std::log10(expMult);
             if (isStandardUnitName(ref)) {
                 multiplier += direction * (logMult + (standardMultiplierList.at(ref) + mult + standardPrefixList.at(pre)) * exp);
-            } else {
+                updated = true;
+            } else if (owningModel(units)) {
                 auto model = owningModel(units);
```

Comment:
I'm not a fan of this sort of thing, calling the same function twice in succession.  The test should be on the `model` object where we can clearly state that we don't want a `nullptr`.